mpas_analysis/ocean/ocean_modelvsobs.py

```diff
@@ -183,14 +183,15 @@ def ocn_modelvsobs(config, field, streamMap=None, variableMap=None):
         outFileLabel = 'sssAquarius'
         unitsLabel = 'PSU'
 
-    ds = xr.open_mfdataset(
-        inputFiles,
-        preprocess=lambda x: preprocess_mpas(x, yearoffset=yearOffset,
-                                             timestr='Time',
-                                             onlyvars=varList,
-                                             selvals=selvals,
-                                             varmap=variableMap))
-    ds = remove_repeated_time_index(ds)
+    ds = openMultifileDataSet(inFiles=inputFiles,
+                              calendar=calendar,
+                              timeVariableName='Time',
+                              variableList=varList,
+                              selectCoordIndices=iselvals,
+                              variableMap=variableMap,
+                              startDate=startDate,
+                              endDate=endDate,
+                              yearOffset=yearOffset)
```

Comment:
Note, this changes public API so we probably want at least a very simple design doc.